src/Symfony/Component/Profiler/README.md

```diff
@@ -0,0 +1,45 @@
+Profiler Component
+==================
+
+Profiler collects information about each request made to your
+HttpKernel-based application and store them for later analysis.
```

Comment:
The Profiler component is now completely independent from the HttpKernel component.